classes/class1/homework/learnyounode/node_modules/bl/.jshintrc

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
Do not push `node_modules` directories to git!
Each user should install them independently to ensure getting the correct version and platform for their dev environment. Also, it clouds your commit history (like this pull request with more than 2,300 files).